data_management_basics/data_management_basics.md

```diff
@@ -0,0 +1,295 @@
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+Sometimes in our work we encounter what we call "data shame".  That's the unpleasant experience of feeling that one's research project is so disorganized that it's embarrassing and needs to be kept hidden.  
+
+If this ever happens to you, or you feel overwhelmed at the prospect of creating a research data management plan, take a breath and realize that most people struggle with the level of organization that a research project entails.  There are lots of reasons why data management is difficult, and you're not alone in wanting to improve.
+
+</div>
+
+### Quiz: Research Data Management
+
+Which of the following is the most comprehensive definition of data management?
+
+[( )] Activities and practices that support long-term storage of data in a secure location.
+[(X)] Activities and practices that support the long-term preservation, access, and use of data.
+[( )] Activities and practices that support making data available on the web.
+[( )] Activities and practices that support cleaning of data after they are collected.
+*****
+<div class = "answer">
+While long term storage of data in a secure location is often a crucial part of research data management (RDM), it's really only one small aspect of the full scope of RDM.  Similarly, cleaning data is almost always a stage in research data management, but it only constitutes part of RDM.  While some research data may need to be made available online, that's not always the case, and so we're once again left with only an incomplete definition of research data management.  The most complete answer is that research data management consists of activities and practices that support the long-term preservation, access, and use of data.
+
+</div>
+*****
+
+## Research Data Lifecycle
+
+At the start of this module, we gave a description of research data management that had two major aspects: work done to keep a research project going, and work done to make research data usable after a project ends.  
+
+We're going to offer a more complex taxonomy of research data management here, based on RDM tasks that belong to each of the five stages in the **research data lifecycle**.  Before we share our visualization of the research data lifecycle, we'd like to offer a caveat.  
+
+> All models are wrong, but some are useful." - George E. P. Box
+
+You've perhaps seen visualizations of the research data lifecycle that show a circular diagram.  We'll add our own circular diagram to the mix, but keep in mind that you may see these stages called slightly different things in different places!  You may also see similar diagrams with more or fewer steps.  We'll go into each phase of research data management in greater detail.
+
+<div style = "background-color:white;">
+
+<script style="display: block" run-once="true" modify="false">
+mermaid.initialize({});
+
+var svg = mermaid.render(
+'approval_flowchart',
```

Comment:
does this fall under the list of images that need an alt text?

Reply:
This isn't an image, actually, it's a diagram of shapes on the html5 canvas, so there's no attached image.  I added an explainer paragraph.